homeassistant/components/hddtemp/sensor.py

```diff
@@ -42,7 +41,7 @@
 PLATFORM_SCHEMA = SENSOR_PLATFORM_SCHEMA.extend(
     {
         vol.Optional(CONF_DISKS, default=[]): vol.All(cv.ensure_list, [cv.string]),
-        vol.Optional(CONF_HOST, default=DEFAULT_HOST): cv.string,
+        vol.Required(CONF_HOST): cv.string,
```

Comment:
I think we should instead raise an issue telling the user why this is needed, and fail setup instead of just breaking it, because that would only lead to frustration